stripe_terminal/example/android/app/build.gradle

```diff
@@ -56,6 +56,7 @@ android {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
```

Comment:
```suggestion
```
Android should automatically read the `proguard-rules.pro` file. This line of code should not be needed